arch/arm/src/common/arm_internal.h

```diff
@@ -106,17 +106,17 @@
 #  define _START_DATA  __sfb(".data")
 #  define _END_DATA    __sfe(".data")
 #else
-#  define _START_TEXT  &_stext
```

Comment:
What about the IAR case? Is the rest of the code still compatible?

Reply:
Any code which use link symbols directly without these macro isn't compatible with IAR, but it isn't a problem introduced by this patch.